bucket/butane.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "0.11.0",
+    "description": "Butane translates Butane Configs into Ignition Configs.",
```

Comment:
```suggestion
    "description": "Fedora CoreOS Config Transpiler, FCCT. Translates Butane configs into Ignition configs.",
```

Reply:
The official description is 
>Butane translates human-readable Butane Configs into machine-readable Ignition Configs.

and FCCT should not be used anymore since the rename ([butane#167](https://github.com/coreos/butane/issues/167).

Reply:
Irrelevant. For searching and reference it should be there

Reply:
I updated the description to make the history of the project clearer but want to point out that the e.g. [Homebrew formula](https://formulae.brew.sh/formula/butane) received a separately tracked name change and the [MacPorts version](https://ports.macports.org/port/butane/summary) has no reference at all because it was introduced afterwards. I guess what I am trying to say is: the description may not be the right place to put this info?